happy/HappyNodeRoute.py

```diff
@@ -490,15 +490,30 @@ def __update_state(self):
         else:
             self.removeNodeRoute(self.node_id, self.to)
 
+    def getNodeRoute(self, node_id, route_type):
+        """This function will get node route base on node_id and route_type.
```

Comment:
nit: typo `based` rather than `base`

Reply:
@jenniexie, thanks for taking a first pass at this. I don't know if there's a Python documentation standard; however, at minimum, I'd expect something Doxygen-like here that includes a @brief description, @param declarations describing each input and output parameter, and a @return/@retval describing the return value(s).

Reply:
fixed